### PR TITLE
Add Status Embark to README

### PR DESCRIPTION
**Pull request by iurimatias**

### CODE REVIEW

**Comment by frangio** [comment]
Very cool! Will get back to this next week.

File: README.md [hunk 1]
```diff
@@ -39,6 +56,15 @@ contract MyContract is Ownable {
 }
 ```
 
+If you are using Embark, you can also import directly from github:
+
+```solidity
+import 'github.com/OpenZeppelin/openzeppelin-solidity/contracts/ownership/Ownable.sol';
```
[the remark this answers is not in the document]
Comment (frangio) [reply README.md]: +1

I like the proposal, but we cannot merge if this import points to `master`.

Additionally we need to test this before merging.

**Comment by iurimatias** [comment]
@elopio @frangio Alright, updated to point to the v1.9.0 tag :)

File: README.md [hunk 2]
```diff
@@ -13,15 +13,32 @@ With OpenZeppelin, you can build distributed applications, protocols and organiz
 
 ## Getting Started
 
-OpenZeppelin integrates with [Truffle](https://github.com/ConsenSys/truffle), an Ethereum development environment. Please install Truffle and initialize your project with `truffle init`.
+OpenZeppelin integrates with [Truffle](https://github.com/ConsenSys/truffle) and [Embark](https://github.com/embark-framework/embark/)
```
Comment (elopio) [inline README.md]: A nit, but this is missing a period at the end of the sentence :)

**Comment by iurimatias** [comment]
@elopio done! :D

**Comment by frangio** [comment]
Thanks @iurimatias!

**Pull request merged**